rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -71,8 +71,8 @@ def __init__(
     def _decode_uniswap_claim(  # pylint: disable=no-self-use
             self,
             tx_log: EthereumTxReceiptLog,
-            decoded_events: List[HistoryBaseEntry],  # pylint: disable=unused-argument
             transaction: EthereumTransaction,  # pylint: disable=unused-argument
+            decoded_events: List[HistoryBaseEntry],  # pylint: disable=unused-argument
```

Comment:
ooof. That's bad. We need to figure out a way to check order of arguments for all these callbacks.